docs/data/migration/migration-data-grid-v7/migration-data-grid-v7.md

```diff
@@ -43,6 +43,19 @@ Below are described the steps you need to make to migrate from v7 to v8.
 
 - The `apiRef.current.resize()` method was removed.
 - The `<GridOverlays />` component is not exported anymore.
+- `gridRowsDataRowIdToIdLookupSelector` was removed. Use `gridRowsLookupSelector` in combination with `getRowId()` API method instead.
+
+  ```tsx
+  // Before
+  const idToIdLookup = gridRowsDataRowIdToIdLookupSelector(apiRef);
+  ...
+  const rowId = idToIdLookup[id]
+
+  // After
+  const rowsLookup = gridRowsLookupSelector(apiRef);
+  ...
+  const rowId = apiRef.current.getRowId(rowsLookup[id])
+  ```
```

Comment:
A diff might be more readable?
```suggestion
  ```diff
  - const idToIdLookup = gridRowsDataRowIdToIdLookupSelector(apiRef);
  - const rowId = idToIdLookup[id]
  + const rowsLookup = gridRowsLookupSelector(apiRef);
  + const rowId = apiRef.current.getRowId(rowsLookup[id])
  ```
```

Reply:
Already aligned, based on the PR description edit you have made :)